server/sonar-web/src/main/js/apps/projects/components/PageHeader.js

```diff
@@ -19,17 +19,26 @@
  */
 // @flow
 import React from 'react';
+import SearchFilterContainer from '../filters/SearchFilterContainer';
 import { translate } from '../../../helpers/l10n';
 
 type Props = {
+  isFavorite?: boolean,
   loading: boolean,
   onOpenOptionBar: () => void,
+  organization?: { key: string },
+  query: { [string]: string },
   total?: number
 };
 
 export default function PageHeader(props: Props) {
   return (
     <header className="page-header">
```

Comment:
What do you think about placing the search input, the total number of results and the "settings" button in one line?

Otherwise I feel there is a lot of empty space:
![image](https://cloud.githubusercontent.com/assets/2317341/26581250/f91e3cb0-453b-11e7-9dec-b06c622710a2.png)